app/react/ServerRouter.js

```diff
@@ -225,7 +230,9 @@ function handleRoute(res, renderProps, req) {
         true
       );
     })
-    .catch(e => handleError(e, { req }));
+    .catch(e => {
+      respondError(res, e);
```

Comment:
This makes the entire function return a resolved promise with `undefined`.  The previous implementation actually returned the result of handleError.  Please make sure this is a desired effect on the function to not have a return.

Reply:
@RafaPolit @txau, this is ok no one is actually expecting any return here.